src/tests_deprecated/behavior/CMakeLists.txt

```diff
@@ -2,8 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-add_subdirectory(shared_tests)
+if (ENABLE_FUNCTIONAL_TESTS)
+    add_subdirectory(shared_tests)
 
```

Comment:
it's behavior tests, why they should depend on `ENABLE_FUNCTIONAL_TESTS`?

Reply:
Fixed